worlds/x2wotc/mods/lwotc/__init__.py

```diff
@@ -0,0 +1,361 @@
+        if not world.item_manager.enable_progressive_item("ProgressiveAdvancedWeaponTechLwotcCompleted"):
+            warning(f"X2WOTC: Failed to enable progressive LWOTC advanced weapon tech for player {world.player_name}")
+    elif "RifleTech" in world.options.progressive_items:
+        if not world.item_manager.enable_progressive_item("ProgressiveRifleTechLwotcCompleted"):
+            warning(f"X2WOTC: Failed to enable progressive LWOTC rifle tech for player {world.player_name}")
+        if not world.item_manager.enable_progressive_item("ProgressiveAdvancedWeaponTechLwotcCompleted"):
+            warning(f"X2WOTC: Failed to enable progressive LWOTC advanced weapon tech for player {world.player_name}")
+
+    # GREMLINs are upgraded from ADVENT Robotics
+    world.item_manager.disable_progressive_item("ProgressiveGREMLINTechCompleted")
+    if "GREMLINTech" in world.options.progressive_items:
+        if not world.item_manager.enable_progressive_item("ProgressiveGREMLINTechLwotcCompleted"):
+            warning(f"X2WOTC: Failed to enable progressive LWOTC GREMLIN tech for player {world.player_name}")
+
+    # Light and Heavy Armor are researches, not PG projects
+    if "ArmorTech" in world.options.progressive_items:
+        if not world.item_manager.enable_progressive_item("ProgressiveLightArmorTechLwotcCompleted"):
+            warning(f"X2WOTC: Failed to enable progressive LWOTC light armor tech for player {world.player_name}")
+        if not world.item_manager.enable_progressive_item("ProgressiveHeavyArmorTechLwotcCompleted"):
+            warning(f"X2WOTC: Failed to enable progressive LWOTC heavy armor tech for player {world.player_name}")
+
+    # Handle option to force early proving ground
+    if world.options.early_proving_ground:
+        del world.multiworld.early_items[world.player][
+            world.item_manager.item_table["AutopsyAdventOfficerCompleted"].display_name
+        ]
+        world.multiworld.early_items[world.player][
+            world.item_manager.item_table["AutopsyAdventTrooperCompleted"].display_name
+        ] = 1
+
+    # Rocket Launcher is a squaddie Technical skill
+    world.loc_manager.disable_location("UseRocketLauncher")
+
+    # Lost corpses are unobtainable
+    world.loc_manager.disable_location("AutopsyTheLost")
+    world.item_manager.disable_item("AutopsyTheLostCompleted")
+    world.loc_manager.disable_location("UseUltrasonicLure")
+
+    # Ammo, heavy weapons and grenades are deterministic
+    world.loc_manager.disable_location("UseExperimentalAmmo")
+    world.loc_manager.disable_location("UseExperimentalGrenade")
+    world.loc_manager.disable_location("UseExperimentalGrenadeMk2")
+    world.loc_manager.disable_location("UseExperimentalHeavyWeapon")
+    world.loc_manager.disable_location("UseExperimentalPoweredWeapon")
+
+    # Force Level increases by off-world reinforcements which requires special handling
+    world.item_manager.disable_item("ForceLevel:1")
```

Comment:
I didn't catch this the first time but this doesn't actually stop the force level trap from being added to the item pool.
```suggestion
    world.item_manager.trap_items.discard("ForceLevel:1")
```